### DISCUSSION TITLE
User went from being searchable/browsable to not

### DISCUSSION THREAD

**Discussion by DeathStalker77** [nicotine-plus/nicotine-plus#2746] [Q&A]
I upgraded from **Nicotine+-3.3.0.dev6-mingw_x86_64 11-03** to **Nicotine+-3.3.0.dev6-mingw_x86_64 11-06** and suddenly a "Buddy" is no longer accessible - no other changes on either side.

Any ideas?  Thanks!

--- DS

**Comment by mathiascode** [comment]
Not accessible in the sense that a connection attempt times out, and an error message is shown in the GUI? In that case, perhaps the user has a closed port, they attempt an indirect connection to your port,and it goes to another running program using the same port (the Windows port behavior I mentioned earlier).

Try temporarily closing other programs using the same port, reconnect to Soulseek, and see if the issue still happens.

**Comment by DeathStalker77** [comment]
No, as in when I go to View Profile or Browse Share/Folder, it says it can't connect, but they ARE online and the files show as QUEUED, but the files don't DL as they previously did.

**Comment by DeathStalker77** [comment]
Well, I dropped back to the 03-NOV build and it is the same - 2 users I used to be able to see (profile+shares), message with, get search results from, I no longer can.  No idea why.  Nothing has changed on my side - and, to my knowledge - nothing on their sides.
